scripts/spot-grid.py

```diff
@@ -27,7 +27,8 @@
 from odemis.util.driver import get_backend_status, BACKEND_RUNNING
 
 MAX_WIDTH = 2000  # px
-PIXEL_SIZE_UM = 3.45 / 50.
+PIXEL_SIZE_SAMPLE_PLANE = 3.45
```

Comment:
Please add a unit as comment (µm?). Possibly, seeing the rest of the changes, it'd be better to store in m here too. (3.45e-6)